doc/admin/config.rst

```diff
@@ -163,6 +167,11 @@ Example::
 ``user``, ``password``, ``host``, ``port``
     Connection details for the database connection. Empty by default.
 
+``sslmode``, ``sslrootcert``
+    Connection TLS details for the PostgreSQL database connection. Possible values of ``sslmode`` are [disable, allow, prefer, require, verify-ca, verify-full]. Possible value of the ``sslrootcert`` is the accessible path of the ca certificate. Both values are empty by default.
+
+``sslmode``, ``sslrootcert``, ``sslcert``, ``sslkey``
```

Comment:
Should we really refer to ``sslmode`` and ``sslrootcert`` here again?

Reply:
No, you're right. I've simplified it.